README.md

```diff
@@ -74,7 +74,7 @@ Here is a list of variables that you can use to customize your newly copied `.en
 | `PHONE_CARRIER` | [Supported carriers](#supported-carriers) for SMS | Email configuration required |
 | `PLAY_SOUND` | Play this sound notification if a card is found | E.g.: `path/to/notification.wav`, relative path accepted, valid formats: wav, mp3, flac, [free sounds available](https://notificationsounds.com/) |
 | `PUSHOVER_TOKEN` | Pushover access token | Generate at https://pushover.net/apps/build |
-| `PUSHOVER_USERNAME` | Pushover username |
+| `PUSHOVER_USER` | Pushover username |
```

Comment:
I think you meant to only include this change. Remove the rest of the changes and we'll be good to go!

Reply:
Other things changed on your repo and I didn't update my fork properly.  I created a new pull request.